### ISSUE TITLE
can you share the script used for generating training dataset ?

### ISSUE THREAD

**Issue by argman** [senlinuc/caffe_ocr#4]
Really nice project btw!

**Comment by senlinuc** [comment]
Yes, you are right, attention-based encoder-decoder should be better than lstm+ctc when modeling the language model.
Generating Chinese dataset is more complicated than you think, but I will share my simplified code soon.

**Comment by zhousteven** [comment]
大神，很像知道你的训练数据如何将背景与字符进行合成，拉伸等操作的？能提供下脚本拜读么？

**Comment by dajidali010** [comment]
@senlinuc 大神啊 能share您字符生成的代码不，谢谢了。。。。。。。。

**Comment by blacklee5** [comment]
什么时候开放数据生成代码 @senlinuc

**Comment by nmwhqjl** [comment]
想测试一下效果编译都不过请帮帮 QQ393777485
